src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade-form/upgrade-start-modal.component.html

```diff
@@ -14,7 +14,8 @@
                         spacingClass="mb-3"
                         *ngIf="showImageField"
                         i18n>Make sure to put the correct image. Passing an incorrect image can lead the cluster into an undesired state.</cd-alert-panel>
-        <div class="form-group row">
+        <div *ngIf="versions !== undefined"
```

Comment:
?
```suggestion
        <div *ngIf="versions"
```

Reply:
We get versions as `undefined` if `info$` observable has error (upgrade listing)

Reply:
@avanthakkar but `if (versions)` should also handle the undefined data's AFAIK, doesn't it?

Reply:
It can be null also right? So i think we need to explicitly check for `undefined`

Reply:
even if its null, that is the same isn't it? because there is no version and we could hide it.

Reply:
okay, well I predicted based on value coming from upgrade component. It cam undefined so I put condition for undefined

Reply:
let's handle both and just go for `if (versions)`.

Reply:
ok cool, will update!